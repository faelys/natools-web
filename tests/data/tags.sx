(root tags)
(tag-elements
   (title "Tag " (current (name (2))))
   (main-text "<p>Pages:</p>\n"
      (current "<ul>\n"
         (all-elements (render "<li>" (element title) "</li>\n"))
         "</ul>\n"))
   (sidebar
      "<ul>\n"
      (current (lesser-children (render "<li>" (name (2)) "</li>\n")))
      "<li style=\"current\">" (current (name (2))) "</li>\n"
      (current (greater-children (render "<li>" (name (2)) "</li>\n")))
      "<ul>\n"))
(index-elements
   (title "All tags")
   (main-text
      (current
         (all-children
            (render
               "<h2>" (name (2)) "</h2>\n<ul>\n"
               (all-elements (render "<li>" (element title) "<li>\n"))
               "</ul>\n"))))
   (sidebar
      (current
         "<ul>\n"
         (all-leaves (render "<li>" (name (2 last)) "</li>\n"))
         "</ul>\n")))
