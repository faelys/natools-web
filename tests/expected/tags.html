<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
  <head>
    <title>All tags</title>
  </head>
  <body>
<h1>All tags</h1>
<h2>topic1</h2>
<ul>
<li>First Page<li>
<li>Third Page<li>
</ul>
<h2>topic2</h2>
<ul>
<li>Second Page<li>
</ul>
<h2>topic3</h2>
<ul>
<li>Second Page<li>
<li>Third Page<li>
</ul>
    <div class="sidebar">
<ul>
<li>Secondary subtopic of topic I</li>
<li>Subtopic of topic I</li>
<li>Topic II</li>
<li>Topic III</li>
</ul>
    </div>
    <div class="footer">
<p>Default footer</p>
    </div>
  </body>
</html>
